### ISSUE TITLE
How to update to v2?

### ISSUE THREAD

**Issue by thematrixdev** [mnakada/atomcam_tools#84]
Hello there.

```
お手数ですがUpdateを２度実行してください
```

What does it mean?
Does it mean copying the files onto the SD-card twice?
Or power cycle twice?

Thanks.

**Comment by mnakada** [comment]
It's mean the WebUI Update button.
If you want to write factory_t31_ZMC6tiIDQN and rootfs_hack.squashfs directly to the SD-Card, you only need to do it once.

**Comment by thematrixdev** [comment]
I have unzipped `atomcam_tools.zip` and put the four files onto the SD card, with everything else deleted.
There are only four files on the SD card.
Rebooting the camera, it does not connect to WIFI.
I have reset the camera.
After scanning the QR code from the official app, the camera speaks "cannot find the network name".
I have changed my WIFI from channel 1 to channel 6, same.
I have tried using WIFI from Pixel 7 phone, same.
Strange.
